Example/Tests/Tests.swift

```diff
@@ -194,4 +194,21 @@ class Tests: XCTestCase {
     XCTAssertEqual(_touches.count, 1)
   }
     
+  func test_removingTouchViewsAfterDisabling() {
+    XCTAssertTrue(_touches.isEmpty)
+    
+    let defaultState = ShowTime.enabled
+    ShowTime.enabled = .always
+    
+    let window = UIWindow()
+    let startEvent = MockEvent(touches: [MockTouch(phase: .began)])
+    window.sendEvent(startEvent)
+    ShowTime.enabled = .never
```

Comment:
Nice test! All I would do differently is assert that the touch was stored properly before setting to `.never`.